content/blog/2020-03-19-howto/index.en.md

```diff
@@ -0,0 +1,81 @@
+
+**Step 1.2.** The YAML header for this article is as follows.
+
+```
+---
+title: How to submit an article
+author: Samuel Perreault and David Beauchemin
+date: '2020-04-21'
+slug: howto-en
+type: post
+categories: ["Contribute"]
+tags: []
+description: "A template to contribute to the blog .Layer"
+featured: "howto-cover.png"
+featuredpath: "img/headers/"
+---
+```
+The fields are almost all *self-explanatory*.
+The `date` field should contain the creation date of the file. We will change it for the publication date in due time (in the *Pull Request*).
+We will also take care of the `categories` and` tags` fields.
+The `slug` field is a *nickname* for your article, which will be used to name the various files linked to the article in the blog directory.
+The `featured` field should contain the name of the cover image file, while` featuredpath` (which should remain unchanged) indicates where to find the file. This is where you should place your cover image.
+Finally, if the article submitted is already published on another platform, please add the `canonical` field to specify the platform. For example, in the header of the article *What's wrong with Scikit-Learn* published on the blog, we find
+```
+canonical: https://www.neuraxio.com/en/blog/scikit-learn/2020/01/03/what-is-wrong-with-scikit-learn.html
+```
+in addition to the mention at the end of the article which refers to the original publication.
+
+
+As for writing the article, you have to know the basics of Markdown.
+In addition to the content already in the *template* that constitutes this article, we recommend this little [*cheatsheet*] (https://github.com/adam-p/markdown-here/wiki/Markdown-Here-Cheatsheet) to use Markdown.
+
+# 2. Submission of an article
+
+**Steps 2.1. and 2.2.** These are classic operations of [Git](https://git-scm.com/). If you are not comfortable with Git, let us know and we will make it up! Otherwise, Atlassian offers great free online [training](https://www.atlassian.com/fr/git).
+
+**Steps 2.3.** Create a new directory/folder `content/blog/YYYY-mm-dd-post-name` (in your new repository created in the previous step) and insert your article and the static files (e.g. images) necessary for its compilation. As mentioned earlier in the article, we ask to stick to the .md (or .html), it is no longer a question of compiling R code, Python, Julia, etc. when the article is included in the blog repository.
+
+**Step 2.4.** Make a *Pull Request* to the *master* branch on the [blog github](https://github.com/dot-layer/blog). Another classic Git operation. From there, contributors who manage the blog will review the article, make constructive recommendations, and ensure that the rendering is *clean*.
+
+
+Uh there ... No more complicated than that. Hope this will help you.
```

Comment:
Hope this helps 
ou 
I / we hope this will help you

Reply:
So there
Uh there c'est pas vraiment utilisé en anglais

Reply:
Aight je change ca haha j'ai p-e traduit un peu trop mot pour mot haha